### ISSUE TITLE
Unable to skip Intro/Black screen

### ISSUE THREAD

**Issue by Redund0** [TeamMoonstorm/Starstorm2#577]
Starstorm worked for me yesterday, but after updating some mods this morning the skip intro button doesn't work and even if I wait through the whole intro it then just blackscreens while music still plays. Able to load in after disabling Starstorm

[LogOutput.log](https://github.com/user-attachments/files/17114504/LogOutput.log)

**Comment by Redund0** [comment]
I have not, I've even deleted the configs to have them re-create fresh with no luck.
Also I remembered I should include my profile code 
01922421-8eba-b4b3-8425-f9e0bf94e2b3

**Comment by Slipskip** [comment]
It's just that there isn't really much else to go on other than

More difficulties is 2 years old
BiggerBazzar from what I remember reading on various discords is also kind of dead as of the update
Both of Nuxlars coolereclipse mods are pre-seekers but they might still just work though the error on loader with coolerclipse has me doubting
And Standalone Majorconstruct is 6 months old but that might also just work since it's just reimplementing an enemy already in the game files

Beyond that all if those low hanging fruit don't solve it is try ss2 on a new profile on it's own and see if that allows it to boot. It could just be cursed profile moment or a mod incompat

**Comment by Redund0** [comment]
Good news, I did some debugging and found the issue.
It's the RealerEliteVariety mod, specifically when some elites are turned off in the config. In my case I had pillaging, Sandstorm, and Tinkerer Turned off. I'll ping Prod in the Ror2 modding discord about it too.

As a side note More difficulites and bigger bazzar work for me. And Major construct seems to. Cooler eclipse stages doesn't work though, so that's a good catch (don't play eclipse much)

**Comment by Slipskip** [comment]
Yup right as I was about to post that I'd managed to find it as well lol
